packages/commonwealth/server.ts

```diff
@@ -23,16 +23,16 @@ import { DatabaseCleaner } from './server/util/databaseCleaner';
 import 'express-async-errors';
 
 // bootstrap production adapters
+if (config.APP_ENV === 'production') {
+  stats(HotShotsStats());
+  analytics(MixpanelAnalytics());
+  if (config.NOTIFICATIONS.FLAG_KNOCK_INTEGRATION_ENABLED)
```

Comment:
Can't do this as it disables notifications in all environments except production so we wouldn't be able to test anywhere. KnockProvider should _only_ be based on `FLAG_KNOCK_INTEGRATION_ENABLED` not `APP_ENV`.

Additionally, filtering by `APP_ENV` prevents using the local StatsD dev server for testing.